css/index.css

```diff
@@ -1 +1,301 @@
+
+}
+
+header nav a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+header .logo {
+    margin: 28px 10.3% 0 10.3%;
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    margin-top: 72px;
+    justify-content: flex-end;
+}
+
+.cta .cta-text {
+    font-size: 7.2rem;
+    text-align: center;
+    letter-spacing: 5px;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    margin: 0 9.31% 0 0;
+}
+
+.cta .cta-text button {
+    border: 1px double black;
+    font-size: 1.6rem;
+    background: white;
+    width: 170px;
+    height: 32px;
+    margin-top: 10px;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 1.6rem;
+   line-height: 18px;
+}
+
+.main-content .top-content {
+    display: flex;
+    font-size: 1.6rem;
+}
+
+.main-content .middle-img {
+    max-width: 100%;
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
+.main-content .bottom-content {
+    display: flex;
+}
+
+.main-content .bottom-content p {
+    padding-right: 20px 2.5%;
+}
+
+/* Contact Styles */
+
+.contact p {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    font-size: 1.6rem;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin-top: 40px;
+    margin-bottom: 20px;
+}
+
+/* Tablet */
+@media (max-width: 800px) {
+
+    .container {
+
+        max-width: 800px;
+    }
+
+    header {
+        flex-wrap: wrap;
+    }
+  
+    header nav {
+        width: 96%;
+        padding: 0 3.75%;
+    }
+
+    header .logo {
+        order: -1;
+        margin: 30px 100% 0;
+    }
+
+    .cta {
+        justify-content: center;
+    }
+    
+    .cta .cta-text {
+        margin: 0;
+    }
+
+    .cta img {
+        display: none;
+    }
+
+    .main-content .middle-img {
+       width: 100%;
+    }
+
+    .main-content {
+       margin: 75px 2.5% 35px;
+    }
+
+    .text-content {
+        margin: 0 1.25%;
+    }
+
+    .contact {
+        margin: 0 3.75%;
+    }
+}
+
+/* Phone */
+@media (max-width: 500px) {
+
+    .container {
+        max-width: 500px;
+    }
+
+    header {
+        justify-content: center;
+    }
+
+    header nav {
+        flex-direction: column;
+        border-top: 1px solid gray;
+        /* margin: 0;
+        padding: 0; */
+    }
+
+    header nav a {
+        width: 100%;
+        text-align: center;
+        padding-top: 20px;
+        padding-bottom: 20px;
+        font-size: 1rem;
+        border-bottom: 1px solid gray;
+    }
+
+    header nav a:hover {
+        background: gray;
+    }
+
+    header .logo {
+        margin-top: 20px;
+        margin-bottom: 20px;
+    }
+
+    .cta {
+        margin-top: 50px;
+        flex-flow: column;
+        align-items: center;
+    }
+
+    .cta img {
```

Comment:
This block isn't necessary since the img has `display: none` on it in the tablet view. The effect will cascade.